pkg/referrerstore/oras/utils_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package oras
+
+import (
+	"testing"
+
+	"github.com/deislabs/ratify/pkg/ocispecs"
+	"github.com/opencontainers/go-digest"
+	oci "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+func TestIsInsecureRegistry(t *testing.T) {
+	testCases := []struct {
+		registry string
+		config   OrasStoreConf
+		expected bool
+	}{
+		{
+			registry: "ghcr.io/test/registry:v0",
```

Comment:
is it possible to add description for each test similar to https://github.com/deislabs/ratify/blob/main/pkg/certificateprovider/azurekeyvault/provider_test.go#L59 ? sometimes its easier to debug the failures knowing which test cases failed.

Reply:
great idea! will update